src/command_modules/azure-cli-maps/azure/cli/command_modules/maps/_params.py

```diff
@@ -34,7 +34,7 @@ def load_arguments(self, _):
         c.argument('sku_name',
                    options_list=['--sku', '-s'],
                    help='The name of the SKU.',
-                   arg_type=get_enum_type(['S0']))
+                   arg_type=get_enum_type(['S0', 'S1']))
```

Comment:
Is there an enum in your SDK for this parameter?  If so, you can simply pass the enum class and not have to manually update the CLI each time.

Reply:
for now, we dont, but thats a good idea, Will change this, when we plan to add the next SKU.